pycloudflare/utils.py

```diff
@@ -0,0 +1,32 @@
+from functools import wraps
+
+from pycloudflare.exceptions import SSLUnavailable
+from pycloudflare.services import HTTPServiceError
+
+
+errors_mapping = {
+    1001: SSLUnavailable,
+}
```

Comment:
I had this idea, but having different mappings for different decorators seems awful to me. I've posted request to CF technical support, waiting for response.